npc/re/instances/saras_memory.txt

```diff
@@ -1133,7 +1133,7 @@ OnDisable:
 	end;
 OnRespawn:
 	.@map$ = instance_mapname("1@sara");
-	monster(.@map$,155,226,"Doyen Irene",2542,1,instance_npcname(strnpcinfo(0))+"::OnMyMobDead");
+	monster(.@map$, 155, 226, _("Doyen Irene"), IRENE_ELDER, 1, instance_npcname(strnpcinfo(0))+"::OnMyMobDead");
 	unittalk($@mobid[0], _("I will have revenge for Sara's Mom!"));
```

Comment:
i think hercules haven't implement the extension for `*monster` script command to support array.
https://github.com/rathena/rathena/commit/2cee5b6#diff-0dec96292c5f60451efa76decb262afaR9424

`$@mobid` are always zero.